addons/purchase_requisition/models/purchase.py

```diff
@@ -259,6 +260,13 @@ class PurchaseOrderLine(models.Model):
     price_total_cc = fields.Monetary(compute='_compute_price_total_cc', string="Company Subtotal", currency_field="company_currency_id", store=True)
     company_currency_id = fields.Many2one(related="company_id.currency_id", string="Company Currency")
 
+    @api.model
+    def formatted_read_group(self, domain, groupby=(), aggregates=(), having=(), offset=0, limit=None, order=None) -> list[dict]:
+        if self.env.context.get('is_comparison_mode'):
+            no_agg_fields = {'price_unit', 'price_subtotal', 'product_qty', 'price_total_cc'}
+            aggregates = [agg for agg in aggregates if agg == '__count' or agg.partition(':')[0] not in no_agg_fields]
```

Comment:
```suggestion
            aggregates = [agg for agg in aggregates if agg == '__count']
```

Keep it simple. Since you've already `removed all those` fields, and the remaining two, `company_currency_id` and `currency_id` are both `column_invisible=True` in the view, they shouldn't be shown anyway. WDYT?